HaxeManual/09-macros.tex

```diff
@@ -298,6 +298,15 @@ \section{Limitations}
 \subsection{Macro-in-Macro}
 \label{macro-limitations-macro-in-macro}
 
+Build macros cannot be invoked from a macro context. This means it is impossible to use a macro to create a class which will provide build macros for other classes.
+
+\since{4.0.0}
+
+Expression macros are also disallowed in a macro context.
```

Comment:
I think we should mention that they only fail (i.e. throw an exception) if they are actually invoked. This is important because it avoids a lot of fencing.